codespell_lib/data/dictionary.txt

```diff
@@ -22445,6 +22445,7 @@ fragmeneting->fragmenting
 fragmenets->fragments
 fragmentization->fragmentation
 fragmnet->fragment
+fram->frame
```

Comment:
Other suggestions: `gram`, `fam`, `dram`, `cram`.

I don't know if there are actually such typos out there.

Reply:
Do you mean fam as in the informal for family?

Reply:
Yes. I guess that would be a very rare typo. Not sure it is worth it.

Reply:
I also wasn't sure if aspell would be happy with it...